__tests__/app.test.js

```diff
@@ -247,3 +247,40 @@ describe("/api/articles/:articles_id/comments", () => {
     });
   });
 });
+
+describe('PATCH /api/articles/:article_id', () =>
+{
+    test('STATUS 200 - Responds with the article object with the updated vote.', () =>
+    { 
+        return request(app)
+            .patch('/api/articles/1')
+            .send({inc_votes: 1})
+            .expect(201)
+            .then(({body: { article }}) =>
+            {
+                expect(article).toEqual(
+                    {
+                      article_id: 1,
+                      title: "Living in the shadow of a great man",
+                      topic: "mitch",
+                      author: "butter_bridge",
+                      body: "I find this existence challenging",
+                      created_at: "2020-07-09T20:11:00.000Z",
+                      votes: 101,
+                      article_img_url:
+                        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                    })
+            });
+    });
+    test("STATUS 400 - Responds with 'Bad Request' due to sent object missing properties.", () =>
+    {
+        return request(app)
+            .patch('/api/articles/1')
+            .send({})
+            .expect(400)
+            .then(({body: { msg }}) =>
+            {
+                expect(msg).toBe('Bad request')
+            });
+    });
+});
```

Comment:
A good start, however, this may be a 200, and I would also like to see tests for the following.


Status 400, invalid ID, e.g. string of "not-an-id"

Status 404, non existent ID, e.g. 0 or 9999

Status 400, incorrect body, e.g. inc_votes property is not a number

Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)